### ISSUE TITLE
Question: Warning in the logs on startup of our pods. io.grpc.StatusRuntimeException: INVALID_ARGUMENT: One or more TimeSeries could not be written: Unrecognized resource type: .: timeSeries[0]

### ISSUE THREAD

**Issue by dh-ger** [GoogleCloudPlatform/opentelemetry-operations-java#57]
Hi,

We are using the java auto opentelemetry instrumentation along with the cloud trace exporter.
Our dockerfile looks like this:
```
FROM <OUR OPENJDK RUNTIME CONTAINER>

COPY pc-api/build/libs/pc-api-0.0.1-SNAPSHOT.jar /app/pc-api-0.0.1-SNAPSHOT.jar
COPY pc-api/jar/opentelemetry-javaagent-all-v0.10.1.jar /app/opentelemetry-javaagent-all-v0.10.1.jar
COPY pc-api/jar/exporter-auto-0.10.1.jar /app/exporter-auto-0.10.1.jar

ENTRYPOINT ["java","-javaagent:/app/opentelemetry-javaagent-all-v0.11.0.jar",  "-Dotel.exporter.jar=/app/exporter-auto-0.10.1.jar", "-Dotel.propagators=tracecontext,baggage,b3multi,b3", "-jar","/app/pc-api-0.0.1-SNAPSHOT.jar"]

```

We found the following warning in the logs of our pods on startup, and are wondering if it is anything that we should be worried about/if there is any way to get rid of it?

```
[opentelemetry.auto.trace 2020-12-03 15:58:15:087 +0000] [IntervalMetricReader_1] WARN io.opentelemetry.sdk.metrics.export.IntervalMetricReader - Exporter threw an Exception
com.google.api.gax.rpc.InvalidArgumentException: io.grpc.StatusRuntimeException: INVALID_ARGUMENT: One or more TimeSeries could not be written: Unrecognized resource type: .: timeSeries[0]
        at com.google.api.gax.rpc.ApiExceptionFactory.createException(ApiExceptionFactory.java:49)
        at com.google.api.gax.grpc.GrpcApiExceptionFactory.create(GrpcApiExceptionFactory.java:72)
        at com.google.api.gax.grpc.GrpcApiExceptionFactory.create(GrpcApiExceptionFactory.java:60)
        at com.google.api.gax.grpc.GrpcExceptionCallable$ExceptionTransformingFuture.onFailure(GrpcExceptionCallable.java:97)
        at com.google.api.core.ApiFutures$1.onFailure(ApiFutures.java:68)
        at com.google.common.util.concurrent.Futures$CallbackListener.run(Futures.java:1039)
        at com.google.common.util.concurrent.DirectExecutor.execute(DirectExecutor.java:30)
        at com.google.common.util.concurrent.AbstractFuture.executeListener(AbstractFuture.java:1165)
        at com.google.common.util.concurrent.AbstractFuture.complete(AbstractFuture.java:958)
        at com.google.common.util.concurrent.AbstractFuture.setException(AbstractFuture.java:749)
        at io.grpc.stub.ClientCalls$GrpcFuture.setException(ClientCalls.java:545)
        at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:515)
        at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:426)
        at io.grpc.internal.ClientCallImpl.access$500(ClientCallImpl.java:66)
        at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:689)
        at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$900(ClientCallImpl.java:577)
        at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:751)
        at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:740)
        at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
        at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:830)
        Suppressed: com.google.api.gax.rpc.AsyncTaskException: Asynchronous task failed
                at com.google.api.gax.rpc.ApiExceptions.callAndTranslateApiException(ApiExceptions.java:57)
                at com.google.api.gax.rpc.UnaryCallable.call(UnaryCallable.java:112)
                at com.google.cloud.monitoring.v3.MetricServiceClient.createTimeSeries(MetricServiceClient.java:1146)
                at com.google.cloud.monitoring.v3.MetricServiceClient.createTimeSeries(MetricServiceClient.java:1088)
                at com.google.cloud.opentelemetry.metric.CloudMetricClientImpl.createTimeSeries(CloudMetricClientImpl.java:24)
                at com.google.cloud.opentelemetry.metric.MetricExporter.createTimeSeriesBatch(MetricExporter.java:161)
                at com.google.cloud.opentelemetry.metric.MetricExporter.export(MetricExporter.java:148)
                at io.opentelemetry.sdk.metrics.export.IntervalMetricReader$Exporter.run(IntervalMetricReader.java:195)
                at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
                at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
                at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
                ... 3 more
Caused by: io.grpc.StatusRuntimeException: INVALID_ARGUMENT: One or more TimeSeries could not be written: Unrecognized resource type: .: timeSeries[0]
        at io.grpc.Status.asRuntimeException(Status.java:533)
        ... 15 more
```

**Comment by jsuereth** [comment]
I believe this is a duplicate of #46

**Comment by jsuereth** [comment]
Also, thanks for the reproduction!  This *should* be fixed in 0.11.0, but we'll be making a more up-to-date release shortly for the Trace 1.0.
